docs/CNVs/CNV-methods.tex

```diff
@@ -482,6 +497,134 @@ \subsection{Using Panel of Normals for Allelic Fraction Model} \label{allelic-Po
 \end{equation}
 %
 where $f$ and $\pi$ may once again be sampled via adaptive Metropolis.
+
+\subsection{Generative Model for Read Counts} \label{coverage-model}
+In this section we address several goals:
+
+\begin{enumerate}
+
+\item Connect copy ratio (or copy number) and raw read counts in a single probabilistic model without transforming the data.
+
+\item Take into account the Poisson nature of coverage depth, thereby giving less weight to low-coverage targets and separating the inherent variance due to Poisson statistics from experimental noise.  We want to use the panel of normals to subtract only the latter.
+
+\item Choose the number of principal components to use in an automatic and principled manner.
+
+\item Use an algorithm that does not waste time calculating all principal components when we only want the few most significant ones.
+
+\item Correct for CNV events that occur in the panel of normals.
+
+\end{enumerate}
+
+Suppose we have vectors of read counts over a set of $T$ targets for $S$ samples, $\vn_s, \, s = 1 \ldots S$ where $n_{st}$ is the coverage of sample $s$ at target $t$.  We imagine that laboratory conditions for a particular sample yielding an underlying bias vector $\vb_s$, where $e^{b_{st}}$ is the propensity of target $t$ to be captured, sequenced, and mapped in the preparation of sample $s$.  Suppose also that sample $s$ has an average depth $d_s$ and a vector of copy numbers $\vc_s$, where the latent variable $c_{st}$ is the copy number of sample $s$ at target $t$.  Our model for coverage is:
+%
+\begin{equation}
+n_{st} \sim {\rm Poisson}(d_s c_{st} e^{b_{st}})
+\end{equation}
+%
+We can achieve many of the goals listed above by performing probabilistic PCA not on $\vn$ directly but rather on $\vb$.  This is one reason why we choose a Poisson parameter proportional to $e^b$ and not simply $b$ -- probabilistic PCA involves a normal distribution on $\vb_s$, whereas the Poisson parameter must be positive.  Furthermore, since PCA is a linear model with additive effects of different latent features, using $e^b$ represents multiplicative effects, which is more plausible.  Our model is
```

Comment:
I don't think we'd be spending a principal component.  Suppose the target-length dependence were exactly linear i.e. we had a perfect analytic model.  In that case it would renormalize the target means (before exponentiation) without affecting the principal components at all, since there are no fluctuations in this effect.  And maybe there is some latent feature that yields variability relating to target length, but we certainly don't know the structure of this relationship a priori.

To use the workhorse GC bias example, it's not actually GC bias per se that needs to be modeled explicitly.  If GC bias were significant but sample-independent it would get absorbed into the target means and a linear model would handle it exactly.  The reason GC needs to be modeled outside of PCA is because each sample has a different non-linear GC response function.  

Now this raises the question of whether maybe every sample has its own target-length bias curve just as it has its own GC bias curve.  If so, we could treat target length like GC.  Regardless, though, I don't think that modeling a sample-independent effect will spare any principal components or yield a more expressive model.